nautobot/utilities/templatetags/helpers.py

```diff
@@ -57,10 +57,13 @@ def hyperlinked_object(value):
     if value is None:
         return placeholder(value)
     display = value.display if hasattr(value, "display") else str(value)
+    element_id = f" id=ipv{value.family}" if hasattr(value, "family") else ""
```

Comment:
HTML tags should be in quotes (so `id="ipv4"`)

Reply:
Instead of introspecting properties of `value`, what about allowing custom HTML properties via optional args (example: https://github.com/django/django/blob/3.2/django/template/defaultfilters.py#L675-L683)

@glennmatthews?

Reply:
> HTML tags should be in quotes (so `id="ipv4"`)

I think that also unquoted attributes are valid W3C syntax for "normal" HTML but you're right: it's recommended to use quotes and to be consistent with you code, it's definitely a better choice. Thanks, Bryan.

> I'm already adding a `field` optional arg to this filter as part of #2616 to solve an issue with `value.display` not always being the way we want to render the value - Django doesn't have a straightforward way to define a filter that takes multiple arguments, that I'm aware of, so maybe this problem needs to be solved in a different way - wonder how difficult it would be to implement an `add_element_id` filter so we could do something like `{{ object|hyperlinked_object|add_element_id:"ipv4" }}` instead?

A neat idea to use another specific filter that would be generic for adding element id. I wanted to come up with some quick fix that would fit its purpose, but this makes more sense and has wider potential at the end of the day.